openmdao/core/implicitcomponent.py

```diff
@@ -71,6 +71,13 @@ def _solve_nonlinear(self):
             else:
                 return result
 
+    def _guess_nonlinear(self):
+        """
+        Provide initial guess for states.
+        """
+        with self._unscaled_context(outputs=[self._outputs], residuals=[self._residuals]):
+            self.guess_nonlinear(self._inputs, self._outputs, self._residuals)
```

Comment:
I was looking at the call coming from newton, but I guess self._system in newton will always be a Group then?